README.md

```diff
@@ -76,6 +77,9 @@ SummaryProcessor accepts the following context in addition to the Processor's co
 
 ```ruby
 {
+  markdown: {
+    footnotes: false # Enable / Disable footnotes in document. (Default: true)
+  },
```

Comment:
Because the SummaryProcessor section has the following description:

> SummaryProcessor accepts the following context in addition to the Processor's context:

... common contexts between `Processor` and `SummaryProcessor` should be described _only_ in the `Processor` section.